src/Trading/components/TradingForm.tsx

```diff
@@ -375,7 +376,16 @@ function TradingForm(props: Props) {
               }
               control={form.control}
               name="manualPrice"
-              rules={{ validate: value => isValidAmount(value) || t<string>("trading.validation.invalid-price") }}
+              rules={{
+                validate: value => {
+                  const valid = isValidAmount(value) && !BigNumber(value).eq(0)
```

Comment:
Does that maybe cause issues initially, before you enter any value?

Reply:
The first validation is triggered after the user clicked on the "Place order" button and the `manualPrice` is set to 0 by default so there should not be any issues. If the user did not manually change the value and the calculation of an optimal price failed, the `manualPrice` will still be 0 and the validation rule will expand the field and return a proper error message.